times_reader/transforms.py

```diff
@@ -64,6 +64,15 @@
     "VDA_FLOP": ["FLOP"],
 }
 
+attr_prop = {
+    "COM_LIM": "LimType",
+    "COM_TSL": "CTSLvl",
+    "COM_TYPE": "Ctype",
+    "PRC_PCG": "PrimaryCG",
+    "PRC_TSL": "Tslvl",
+    "PRC_VINT": "Vintage",
+}
```

Comment:
`attr_prop` is not used anywhere?

Reply:
True! This is a mapping between column names in `FI_Process` and `FI_Comm`, and the attributes. Will make a note to remember to use it when processing information read from `INS-TXT`. Thanks!